lego/apps/events/migrations/0031_event_groups_with_view_permission.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.13 on 2021-03-16 17:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("users", "0025_user_phone_number"),
```

Comment:
See https://github.com/webkom/lego/commit/19cd30e01883a33c2d9153d63b8e9a9454a2277f, This should be updated to the latest migration.